src/acoupi/system/programs.py

```diff
@@ -168,6 +174,59 @@ def write_program_file(
     )
 
 
+def get_program_store(settings: Settings) -> types.Store:
+    """Get the store for the program."""
+    db_path = Path(
+        json.load(settings.program_config_file.open())["paths"]["db_metadata"]
+    )
+    store = SqliteStore(db_path=db_path)
+    return store
+
+
+def end_program(settings: Settings) -> None:
```

Comment:
@mbsantiago Add the end_program() in programs.py file as it has to do with shutting down the program, but let me know if you think there is a better place for this function.

Reply:
I'm wondering if the `system` module is the ideal location for it.  Ideally, functions in `system` should be program-agnostic, working with any program regardless of its configuration schema.  This function currently makes a couple of assumptions:

- It assumes a specific structure for the configuration.
- It assumes the program uses a SQLite store.

While we encourage using the `BasicProgram` template, it's possible to build a program without a store directly from `AcoupiProgram`.  In those cases, this function might not work as expected.